staff/views/orgs.py

```diff
@@ -64,8 +65,11 @@ def form_valid(self, form):
         return HttpResponseClientRedirect(url)
 
     def get_context_data(self, **kwargs):
+        f = furl(reverse("staff:org-create"))
+        f.args.update(self.request.GET)
```

Comment:
I am unclear what this call to update() does.  What is self.request.GET? 

Worth a comment to explain? Or else the reader needs to be familiar with both furl and django specifics, perhaps?

Reply:
oh, GET is the dict of original request querystring args. Ok, I was being dumb

Reply:
This is absolutely me being deeply familiar with both those things, I'll add a comment to explain what we're doing here 👍